src/NUnitTestAdapter/Properties/AssemblyInfo.cs

```diff
@@ -21,5 +21,5 @@
 [assembly: ComVisible(false)]
 
 [assembly: Guid("c0aad5e4-b486-49bc-b3e8-31e01be6fefe")]
-[assembly: AssemblyVersion("3.5.0")]
-[assembly: AssemblyFileVersion("3.5.0")]
+[assembly: AssemblyVersion("3.5.1")]
```

Comment:
I asked because the v2 format docs do not say that and because it never worked that way before. We had to make an entry for each sku in the past. Note also, that it has to work for people who __only__ have dev11 or dev12 installed, so that they are using the installer from that release.